staging/src/k8s.io/apiserver/pkg/admission/attributes.go

```diff
@@ -57,6 +58,13 @@ func (record *attributesRecord) GetNamespace() string {
 }
 
 func (record *attributesRecord) GetName() string {
```

Comment:
@deads2k I know you had thoughts about ordering of admission relative to generated name, which this does not address